### PR TITLE
fixed tinting issues

### PR DESCRIPTION
**Pull request by MillerApps**
I am not a fan of this solution. Feel free to make any changes that you think would improve this **"hack"**! 

Here is what it looks like in the app:
image

### CODE REVIEW

File: Trust/Transfer/Coordinators/RequestCoordinator.swift [hunk 1]
```diff
@@ -51,6 +51,12 @@ class RequestCoordinator: Coordinator {
             applicationActivities: nil
         )
         activityViewController.popoverPresentationController?.barButtonItem = sender
+        activityViewController.setActivityCompletion {
```
Comment (vikmeup) [inline Trust/Transfer/Coordinators/RequestCoordinator.swift]: did you consider reusing this functionality?

**Comment by vikmeup** [comment]
@MillerApps does it have any side effects?

File: Trust/Transfer/Coordinators/RequestCoordinator.swift (hunk 1 above)
Comment (MillerApps) [reply Trust/Transfer/Coordinators/RequestCoordinator.swift]: yes, it didn't work as intended.

**Comment by MillerApps** [comment]
@vikmeup as far I as can tell no.

**Comment by vikmeup** [comment]
Let's create `ActivityViewController ` that implements that functionality

**Comment by MillerApps** [comment]
Okay, I'll do it tomorrow.

**Comment by MillerApps** [comment]
I won't have time to do this today.

**Comment by vikmeup** [comment]
@MillerApps That’s like 2 minute fix. I can take on this

**Comment by MillerApps** [comment]
@vikmeup Okay, sounds good!

File: Trust/Extensions/UIActivityController.swift [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// Copyright SIX DAY LLC. All rights reserved.
+
+import UIKit
+
+extension UIActivityViewController {
+    func setActivityCompletion(completion: @escaping (() -> Void)) {
+        self.completionWithItemsHandler = { _, completed, _, _ in
+            if completed || !completed {
```
Comment (vikmeup) [inline Trust/Extensions/UIActivityController.swift]: `completed || !completed ` this covers all cases, is this still necessary?

File: Trust/Browser/ViewControllers/BrowserViewController.swift [hunk 3]
```diff
@@ -249,6 +249,12 @@ class BrowserViewController: UIViewController {
         let controller = makeShareController(url: url)
         controller.popoverPresentationController?.sourceView = navigationController.view
         controller.popoverPresentationController?.sourceRect = navigationController.view.centerRect
+        controller.setActivityCompletion {
+            self.navigationController?.navigationBar.barTintColor = Colors.darkBlue
```
Comment (vikmeup) [inline Trust/Browser/ViewControllers/BrowserViewController.swift]: can you add this colors to `AppStyle` ?

**Comment by MillerApps** [comment]
@vikmeup I’ll get it done today when I get a chance. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: Trust/Browser/ViewControllers/BrowserViewController.swift (hunk 3 above)
Comment (MillerApps) [reply Trust/Browser/ViewControllers/BrowserViewController.swift]: done

File: Trust/Extensions/UIActivityController.swift (hunk 2 above)
Comment (MillerApps) [reply Trust/Extensions/UIActivityController.swift]: removed, didn't seem to affect anything.

**Pull request closed without merging**